src/pages/documentation/platform-assurance-sdk/tabs/index.md

```diff
@@ -199,7 +194,14 @@ When using React Native, register AEP Assurance with Mobile Core in native code
 
 When using Flutter, register AEP Assurance with Mobile Core in native code as shown on the Android and iOS tabs. --->
 
-<Variant platform="ios" task="implement-assurance" repeat="12"/>
+
+<Variant platform="android" task="implement-assurance" repeat="1"/>
+
+Deep link is the recommended way of connecting to an Assurance session when using the Android SDK. Assurance SDK on Android is already setup to handle incoming intents to your app. It is sufficient to [add an intent filter for incoming links in your app](https://developer.android.com/training/app-links/deep-linking) to complete the deep link configuration. The combination of `android:host` and `android:scheme` (in the form of `<host>://<scheme>`) of this intent filter will serve as the Base URL while creating a session in the [Adobe Experience Platform Assurance UI](https://experience.adobe.com/assurance)
+
+<Variant platform="ios" task="implement-assurance" repeat="13"/>
+
+The [startSession](./assurance-api-reference.md#startsession) API needs to be called to begin an Adobe Experience Platform Assurance session. You should call this API when the app launches with a URL (see code snippet below for sample usage). When called, SDK displays a PIN authentication overlay to begin a session. 
```

Comment:
I think this link should just be `[startSession](#startsession)` because of the weirdness of how tabs/variants work.

Reply:
The `startSession` header is actually defined in the API reference so `[startSession](#startsession)` does not resolve to the right location because when clicked from index it resolves to /documentation/platform-assurance-sdk/#startsession. 

This should have been `[startSession](./api-reference.md#startsession)`  . I corrected the link in the latest commit.

Reply:
Oooh got it. Yup, that's correct.